lib/hanami/helpers/form_helper/form_builder.rb

```diff
@@ -1206,8 +1206,9 @@ def select(name, values, attributes = {}) # rubocop:disable Metrics/AbcSize, Met
           super(attributes) do
             option(prompt) unless prompt.nil?
 
+            already_selected = nil
             values.each do |content, value|
-              if _select_option_selected?(value, selected, _value(name), attributes[:multiple])
+              if (attributes[:multiple] || !already_selected) && already_selected = _select_option_selected?(value, selected, _value(name), attributes[:multiple])
```

Comment:
WDYT, maybe will better (for understanding) replace this condition to separate variable or method?